lib/src/rocket.rs

```diff
@@ -344,7 +345,10 @@ impl Rocket {
     /// Creates a new `Rocket` application using the supplied custom
     /// configuration information. The `Rocket.toml` file, if present, is
     /// ignored. Any environment variables setting config parameters are
-    /// ignored. If `log` is `true`, logging is enabled.
+    /// ignored. If `log` is `false` or the config has `log_output` set to
+    /// `LoggingOutput::Disabled`, the default log implementation is disabled,
+    /// allowing you to use/write one of your own. (`log` is kept around
```

Comment:
I think having both a parameter and a config option that can disable logging is confusing. I would rather it be just the config option, but that definitely is a breaking change.